portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,11 +1,26 @@
 package com.google.sps.data;
 
-public class Comment {
-  private String text;
-  private String name;
+import com.google.auto.value.AutoValue;
 
-  public Comment(String text, String name) {
-    this.text = text;
-    this.name = name;
+
+@AutoValue
+public abstract class Comment {
+  
+  public abstract String name();
+  public abstract String text();
+  public abstract long timePosted();
```

Comment:
nit: you may consider using a java instant instead. not necessary for this commit though.